src/display/canvas.js

```diff
@@ -1280,9 +1282,47 @@ class CanvasGraphics {
     // transparent canvas when we have blend modes.
     const width = this.ctx.canvas.width;
     const height = this.ctx.canvas.height;
-
+    this.defaultBackgroundColor = background || "#ffffff";
     this.ctx.save();
-    this.ctx.fillStyle = background || "rgb(255, 255, 255)";
+
+    if (this.foregroundColor && this.backgroundColor) {
+      this.ctx.fillStyle = this.foregroundColor;
+      const fg = (this.foregroundColor = this.ctx.fillStyle);
+      this.ctx.fillStyle = this.backgroundColor;
+      const bg = (this.backgroundColor = this.ctx.fillStyle);
+
+      if (fg === "#000000" && bg === "#ffffff") {
+        this.foregroundColor = this.backgroundColor = null;
```

Comment:
Will this path be taken by default?

Reply:
Yes. In general I try to put the likely branch in first... old habit...
They should be the two default values.
In settings it's possible to overwrite them or use system ones:
![image](https://user-images.githubusercontent.com/5641725/166708264-8d2d213e-deb9-43e8-9363-2599faed3f39.png)

So in case these values are the default ones, we just use the colors are they are defined in the pdf.